VideoLocker/src/main/java/org/edx/mobile/user/Account.java

```diff
@@ -0,0 +1,201 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Account {
+    @SerializedName("username")
+    @NonNull
```

Comment:
I added annotations to match the API documentation. If they are wrong, then there is an API bug that should be fixed.

Even if we initialized with non-null values, the API could make them null again by return `null` for that field, so that wouldn't be fool-proof.